doc/devel/coding_style_guideline.rst

```diff
@@ -35,6 +35,25 @@ compliance of your code with PEP8. Beyond the aspects checked, as a
 contributor to DIPY, you should try to ensure that your code, including
 comments, conform to the above principles.
 
+-------------------
+Cython coding style
+-------------------
+DIPY recommends the use of the standard Python
+`PEP8 <https://www.python.org/dev/peps/pep-0008/>`_ style when writing
+`Cython <https://cython.org/>` code. The sole exception to these guidelines is
+the line length limitation recommendation: a line length of 120 characters is
+accepted for DIPY Cython code.
+
+Cython-specific syntax should follow these additional rules:
+
+Imports
+-------
+The `cimport`s should add the `c` prefix to the usual Python import package
+shorthand, e.g.::
+
+  cimport numpy as cnp
```

Comment:
Maybe add some explanation?  On the lines of "Adding 'c' to the import line makes it clearer when you are referring to the Cython / C symbols compared to the Python attributes.

Reply:
Done in  4c13254.